keras_nlp/models/bert.py

```diff
@@ -285,30 +305,58 @@ def BertBase(name=None, trainable=True):
     or classification task networks.
 
     Args:
+        weights: String, optional. Name of pretrained model to load weights.
+            Should be one of {names}.
+            If None, model is randomly initialized. Either `weights` or
+            `vocabularly_size` must be specified, but not both.
+        vocabulary_size: Int, optional. The size of the token vocabulary. Either
+            `weights` or `vocabularly_size` must be specified, but not both.
         name: String, optional. Name of the model.
         trainable: Boolean, optional. If the model's variables should be
             trainable.
 
     Example usage:
     ```python
     # Randomly initialized BertBase encoder
-    encoder = keras_nlp.models.BertBase()
+    encoder = keras_nlp.models.BertBase(vocabulary_size=10000)
 
      # Call encoder on the inputs.
-    input_data = {
+    input_data = {{
         "input_ids": tf.random.uniform(
             shape=(1, 512), dtype=tf.int64, maxval=encoder.vocabulary_size),
         "segment_ids": tf.constant(
             [0] * 200 + [1] * 312, shape=(1, 512)),
         "input_mask": tf.constant(
             [1] * 512, shape=(1, 512)),
-    }
+    }}
     output = encoder(input_data)
     ```
-    """
+"""
+
+
+def BertBase(weights=None, vocabulary_size=None, name=None, trainable=True):
+
+    if (vocabulary_size is None and weights is None) or (
+        vocabulary_size and weights
+    ):
+        raise ValueError(
+            "One of `vocabulary_size` or `weights` must be specified "
+            "(but not both). "
+            f"""Received: weights={weights}, """
```

Comment:
Why are you triple quoting f strings? That's not the conversion in this repo or elsewhere in keras

Reply:
Fixed